e2e/tests/api-driven/src/permissions/teamAdmin.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Testing Permissions for teamAdmin Role
+
+  @regression @team-admin-permissions
+  Scenario Outline: teamAdmin permissions
+    Given a teamAdmin is a member of a team
+    When they perform "<ACTION>" on "<TABLE>"
+    Then they have access
+
+    Examples:
+      | TABLE   | ACTION          |
```

Comment:
I did consider adding a `DATA` and `COLUMNS` column here and templating out standard CRUD GraphQL queries. 

At least initially hardcoding queries is simpler (if slightly more tedious).